registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -2464,6 +2464,39 @@
           "tokenManagerType": "mintBurn"
         }
       ]
+    },
+    "0x526c3812e992e0a293ca6f7214f23625acb240e7132eb98e36c11d1428f5a961": {
+      "tokenId": "0x526c3812e992e0a293ca6f7214f23625acb240e7132eb98e36c11d1428f5a961",
+      "deployer": "0x7270b20603FbB3dF0921381670fbd62b9991aDa4",
+      "originalMinter": null,
+      "prettySymbol": "CFG",
+      "decimals": 18,
+      "originAxelarChainId": "centrifuge",
```

Comment:
isn't this token native to Ethereum?

Reply:
The canonical chain is Centrifuge Chain, but we are not able to deploy it on ITS there yet because it's pending the ITS Hub deployment.

Reply:
from the POV of ITS, this token that is created lives on Ethereum (ITS won't know about the other bridge mechanism that was used to create this token from Centrifuge). Can you make that change accordingly?

Reply:
Ok, updated @canhtrinh

Reply:
One question: what impact does setting this value have on ITS?

Wondering about when ITS Hub on Centrifuge Chain goes live and this value is changed to `centrifuge`, whether this would lead to any issues.